src/map/packets/currency1.cpp

```diff
@@ -26,12 +26,26 @@
 #include "entities/charentity.h"
 #include "utils/charutils.h"
 
+// TODO: implement the packet wholesale
+// from https://github.com/atom0s/XiPackets/tree/main/world/server/0x0113
+// clang-format off
+struct bitPackedCurrency1
```

Comment:
I dont think you need to turn off clang format for this

iirc from other PRs I did (purely from memory) it wants the bits not aligned for some reason

``` 
   uint64_t bloodshed_plans_stored : 9;
    uint64_t umbrage_plans_stored : 9;
    uint64_t ritualistic_plans_stored : 9;
    uint64_t tutelary_plans_stored : 9;
    uint64_t primacy_plans_stored : 9;
    uint64_t unused : 19;
```

but that said i dont really care either way since alignment is fine